bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "echo-angular",
-  "version": "1.1.0",
+  "version": "2.0.0",
```

Comment:
Does that mean that the logging config is passed into the logger service as is from the echo service? If so, I think that should have a sensible default. Seems a bit OTT to bump this a major version just because of one of it's dependencies has.